docs/hooks.md

```diff
@@ -152,3 +152,42 @@ Filters the delay for scheduled project updates.
 
 * `$delay`: Delay in minutes. Default is 3 minutes.
 * `$project`: The current project.
+
+----
+
+### `traduttore.loader`
+
+**Since:** 3.0.0
+
+Filters the loader instance for a given repository and project.
```

Comment:
Initially I was not sure what the loader instance ist or what the loader class does. Maybe add something like that it is to do with the VCS that is used for the repository.